Exercises/lesson19/index.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+const BIRTHMONTH = '13-03-2008';
+const FIRSTNAME = 'Anser';
+const LASTNAME = 'Khan';
+let city, age, nextCity;
+city = "Karachi";
+age = "12";
```

Comment:
If it's a number, you shouldn't enclose the value in quotes. If you do, then it will be a string

`age = 12;`